packages/cli/test/integration.js

```diff
@@ -2635,7 +2635,7 @@ test('next unsupported functions config shows warning link', async t => {
   t.is(output.exitCode, 0, formatOutput(output));
   t.regex(
     output.stderr,
-    /Ignoring function property `runtime`\. When using Next\.js, only `memory`, `maxDuration`, and `cron` can be used\./gm,
+    /Ignoring function property `runtime`\. When using Next\.js, only `memory`, and `maxDuration` can be used\./gm,
```

Comment:
I think this needs to change in the API first before you update this test

Reply:
is there a test failing because of this already?

Reply:
Yes this test is failing as you can see. You'll need to change it back for tests to pass

https://github.com/vercel/vercel/actions/runs/4187893545/jobs/7258336358#step:8:1134

Reply:
(I am fine reverting, I am just wondering which test is currently not passing because of this)

Reply:
Done